_docs/proxying.md

```diff
@@ -158,6 +158,21 @@ stubFor(get(urlEqualTo("/users/12345.json"))
   .withStatus(503)));
 ```
 
+Also, we can enable/disable pass through unmatched requests to the target indicated by the original requests by enabling/disabling proxyPassThrough flag.
+By default, flag is set to true. This flag can be enabled/disabled either by passing CLI option while running jar as described in [Running Standalone](/docs/running-standalone/#command-line-options) or by updating global settings.
+
+Update proxyPassThrough flag via Java client
+```java
+WireMockServer wireMockServer = new WireMockServer(options().proxyPassThrough(false));
+```
```

Comment:
Can we also show how to do this at runtime in Java i.e. by updating the `GlobalSettings`?

Reply:
And explain that the `options()` approach is for startup, where as the `GlobalSettings` allows it to be changed without a restart.

Reply:
incorporated